docs/tutorials/forum-application/9.run-app.md

```diff
@@ -0,0 +1,261 @@
+# Run Application
+
+In previous sections you learned about different ABCI 2.0 methods and how they are used.
+
+In this section you will learn how to run the `Forum Application`.
+
+## Initializing and Running
+
+The application is almost ready to run, but first we'll need to populate the CometBFT configuration file.
+
+The following command will create a cometbft home directory in your project and add a basic set of configuration
+files in ~/config/.
+
+For more information on what these files contain see the [configuration documentation](https://docs.cometbft.com/v1.0/references/config/).
+
+### Install comebft
+
+Clone the `cometbft` repository:
```

Comment:
When we release `v1` we could point to that link. Right now we tell them to install `v1.0.0-rc1` because the code is not compatible with `v0.38` (mostly because of ABCI methods/types rename). That's why we are just back porting this to `v1.x`. In the future if needed we could backport to `v0.38.x` but that would need some refactoring to adapt the example code. So I will not point to the docs for now, but in the future it's a good idea. Thanks.